src/pdn/README.md

```diff
@@ -322,62 +347,58 @@ add_pdn_connect -grid rotated_rams -layers {metal4 metal6}
 add_pdn_connect -grid rotated_rams -layers {metal6 metal7}
 ```
 
-### Build power grid
-
-Build a power grid in accordance with the information specified.
-
-```
-pdngen [-skip_trim] \
-       [-dont_add_pins] \
-       [-reset] \
-       [-ripup] \
-       [-report_only] \
-       [-failed_via_report file]
-```
-
-#### Options
-
-| Switch Name | Description |
-| ----- | ----- |
-| `-skip_trim` | Skip the metal trim step, which attempts to remove metal stubs |
-| `-dont_add_pins` | Prevent the creation of block pins during |
-| `-reset` | Reset the grid and domain specifications |
-| `-ripup` | Ripup the existing power grid, as specified by the voltage domains |
-| `-report_only` | Print the current specifications |
-| `-failed_via_report` | Generate a report file which can be viewed in the DRC viewer for all the failed vias (ie. those that did not get built or were removed). |
-
-### Repairing power grid vias after detailed routing
+### Repairing Power Grid vias after Detailed Routing
 
 To remove vias which generate DRC violations after detailed placement and routing use `repair_pdn_vias`.
 
-```
-repair_pdn_vias [-all] \
-                [-net net_name]
+```tcl
+repair_pdn_vias 
+    [-all]
+    [-net net_name]
 ```
 
 #### Options
 
-| Name | Description |
+| Switch Name | Description |
 | ----- | ----- |
-| `-all` | Repair vias on all supply nets |
-| `-net` | Repair only vias on the specified net |
+| `-all` | Repair vias on all supply nets. |
+| `-net` | Repair only vias on the specified net. |
 
-### Converting former PDNGEN configuration file to tcl commands
+### Converting former PDNGEN configuration file to Tcl commands
 
 To get an initial conversion from the former PDNGEN configuration file to the current tcl commands use `convert_pdn_config`.
 This command will provide an initial set of commands based on the provided file, it is recommended that the user double-check
 the conversion to ensure nothing was missed.
 
-```
+```tcl
```

Comment:
this should not be documented as it's deprecated and will be removed soon and is not supported